impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -490,7 +488,54 @@ void RenderPassGLES::ResetGLState(const ProcTableGLES& gl) {
     }
   }
 
+  if (pass_data.resolve_attachment &&
+      pass_data.resolve_attachment != pass_data.color_attachment &&
+      !is_default_fbo) {
+    // Perform multisample resolve via blit.
+    // Create and bind a resolve FBO.
+    GLuint resolve_fbo;
+    gl.GenFramebuffers(1u, &resolve_fbo);
```

Comment:
Is this generating and deleting a frame buffer each frame?

Reply:
for offscreen fbos we always generate and delete the fbo AFAIK.